backend/app/api/ws/device_namespace.py

```
@@ -0,0 +1,942 @@
+
+            task_status = (
+                "FAILED" if subtask.status == SubtaskStatus.FAILED else "COMPLETED"
+            )
+            try:
+                task_kinds_service.update_task(
+                    db=db,
+                    task_id=subtask.task_id,
+                    obj_in=TaskUpdate(status=task_status),
+                    user_id=user_id,
+                )
+            except Exception as e:
```

Ruff (0.14.14)

Do not catch blind exception: `Exception`

(BLE001)